airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -263,6 +265,20 @@ def transform_mssql(config: Dict[str, Any]):
         }
         return dbt_config
 
+    @staticmethod
+    def transform_clickhouse(config: Dict[str, Any]):
+        print("transform_clickhouse")
+        # https://docs.getdbt.com/reference/warehouse-profiles/clickhouse-profile
+        dbt_config = {
+            "type": "clickhouse",
+            "host": config["host"],
+            "port": config["port"],
```

Comment:
Should this be changed to hardcode 9000 instead?

see this thread in slack: https://airbytehq.slack.com/archives/C01MFR03D5W/p1641461068011100